src/Workspaces/Core/Portable/Execution/Serializer.cs

```diff
@@ -52,7 +52,7 @@ public Checksum CreateChecksum(object value, CancellationToken cancellationToken
         {
             var kind = value.GetWellKnownSynchronizationKind();
 
-            using (Logger.LogBlock(FunctionId.Serializer_CreateChecksum, kind, cancellationToken))
+            using (Logger.LogBlock(FunctionId.Serializer_CreateChecksum, kind.ToString(), cancellationToken))
```

Comment:
creating string out of enum is expansive. put kind in and put static lamdba of k => k.ToString() so that it only run when needed.